src/search.js

```diff
@@ -27,7 +32,7 @@ export function addNormalizedDescription(propertyName, objects) {
   return map(
     object => assoc(
       'normalizedDescription',
-      has(propertyName, object)
+      object[propertyName]
```

Comment:
> I'm not totally OK with this change: the test is about the presence of the key. Using `has` is more explicit about the intention than trying to access the key, which makes me wonder: will it raise an exception? Oh ... no, it's JavaScript, it will be... `undefined` ... or `null` ? On the other side, `has` returns a `boolean` which is reassuring!